internal/move2kubeapi/move2kubeapi.go

```diff
@@ -185,7 +185,16 @@ func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 	artifactName := name + "_" + strconv.FormatInt(t.Unix(), 10)
 	log.Infof("Artifact Name:%s", artifactName)
 
-	err := m2kapp.Translate(name, artifactName, plan)
+	keys, ok := r.URL.Query()["debug"]
+	var err error
+	var debugFlag string
```

Comment:
Do we want to convert it to a boolean here itself?

Reply:
Yes sure, I have converted it to a boolean.